scene/gui/tab_container.cpp

```diff
@@ -785,23 +785,25 @@ TabContainer::TabAlign TabContainer::get_tab_align() const {
 	return align;
 }
 
-void TabContainer::set_tabs_visible(bool p_visibe) {
+void TabContainer::set_tabs_visible(bool p_visible) {
 
-	if (p_visibe == tabs_visible)
+	if (p_visible == tabs_visible)
 		return;
 
-	tabs_visible = p_visibe;
+	tabs_visible = p_visible;
 
 	Vector<Control *> tabs = _get_tabs();
 	for (int i = 0; i < tabs.size(); i++) {
 
 		Control *c = tabs[i];
-		if (p_visibe)
+		if (p_visible)
 			c->set_margin(MARGIN_TOP, _get_top_margin());
 		else
 			c->set_margin(MARGIN_TOP, 0);
 	}
+
 	update();
+	minimum_size_changed();
```

Comment:
Does anyone know if this is a standard code practice? Without `minimum_size_changed()`, the size doesn't seem to update (as I would assume would happen with `update()`)

Reply:
`update` is a member of `CanvasItem`, while `minimum_size_changed` is a member of `Control`, so `update` can't call `minimum_size_changed`.